src/Models/ElementalArea.php

```diff
@@ -198,6 +198,9 @@ public function getOwnerPage()
 
         if ($this->OwnerClassName) {
             $class = $this->OwnerClassName;
+            if (!ClassInfo::exists($class)) {
+                return null;
+            }
```

Comment:
```suggestion
        if ($this->OwnerClassName && ClassInfo::exists($this->OwnerClassName)) {
            $class = $this->OwnerClassName;
```

This way it still falls back to the alternate logic below.